src/containers/LoadScreen.js

```diff
@@ -1,9 +1,12 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+import { push } from 'react-router-redux';
 import { compose } from 'redux';
-import { Spinner } from '../ui/components';
+
+import { Icon, Spinner } from '../ui/components';
 import Fade from '../ui/components/Transitions/Fade';
+import { actionCreators as sessionActions } from '../ducks/modules/session';
 
 const minimumTimeToDisplaySpinner = 1000;
```

Comment:
To test, start a new or restart a session. When the loading screen displays, clicking the "x" in the top right should send you back to the starting screen. To increase your chances of clicking on the "x", you can either load a larger protocol, or increase the `minimumTimeToDisplaySpinner` so you have a longer time to try to get to the cancel button.